### ISSUE TITLE
Improve pass phrase message

### ISSUE THREAD

**Issue by Jerbell** [FlowCrypt/flowcrypt-browser#4024]
![Message](https://user-images.githubusercontent.com/5285606/135198557-c545e817-c0f7-4853-b7f0-8161ecb90bc8.png)

It might be just me, but it's not clear this is the PGP key password as opposed to your Gmail password.  Perhaps just adding in PGP there somewhere?

**Comment by tomholub** [comment]
How about `Enter encryption pass phrase` ?

**Comment by tomholub** [comment]
I suppose we don't have to tell them the reason. `to sign email` could be confused as `to sign in` which suggests Gmail password.

**Comment by Jerbell** [comment]
`Enter encryption pass phrase` would be better I think.

**Comment by tomholub** [comment]
If we wanted to be more technical, then `Enter OpenPGP pass phrase` or `Enter S/MIME pass phrase`.

But a lot of our users may have no clue what these mean, which may make them feel stupid and mildly unhappy :-)

**Comment by tomholub** [comment]
But they'd be the clearest

**Comment by Jerbell** [comment]
I wouldn't do S/MIME for sure.

`Enter FlowCrypt OpenPGP pass phrase`?

**Comment by Jerbell** [comment]
If your users get confused by `Enter OpenPGP pass phrase` then I'm not sure how to differentiate between the email password and the PGP key password.

**Comment by tomholub** [comment]
> I wouldn't do S/MIME for sure.
> 
> `Enter FlowCrypt OpenPGP pass phrase`?

We support both OpenPGP and S/MIME these days, so in some situation it would say OpenPGP, and in some it would be S/MIME.

So I guess it would be one of:
 - `Enter FlowCrypt OpenPGP pass phrase`
 - `Enter FlowCrypt S/MIME pass phrase`

Depending on the key.

**Comment by Jerbell** [comment]
Yes - `Enter FlowCrypt pass phrase` might be the best, if it's not always PGP.  I guess you could always do one of those pop-up `?` which if you hover over it, would explain that it's your OpenPGP or S/MIME key password.

**Comment by martgil** [comment]
I also find it really helpful if it has that short description about passphrases through the help of the tooltip.
